samples/generated/express-embedded-sign-in-widget/package.json

```diff
@@ -7,8 +7,8 @@
     "node": ">=10.0.0"
   },
   "scripts": {
-    "start": "node ./web-server/server.js",
-    "dev": "nodemon ./web-server/server.js --watch ../../../build/cjs"
+    "start": "SIW_VERSION=6.0.1 node ./web-server/server.js",
```

Comment:
How about

```
SIW_VERSION=${SIW_VERSION-6.0.1}
```

so that it will have a default value but also respect existing environment variable

Reply:
This file is overwrote by template, and template always load latest version from registry.